index.html

```diff
@@ -1,383 +1,416 @@
-          sign-ups required. has been the industry's standard dummy text ever
-          since the 1500s, when an unknown printer took a standard dummy text.
-        </p>
-        <ul>
-          <li>
-            <a href="#">CSS</a>
-          </li>
-          <li>
-            <a href="#">HTML</a>
-          </li>
-          <li>
-            <a href="#">BOOTSTRAP</a>
-          </li>
-          <li>
-            <a href="#">RUBY</a>
-          </li>
-        </ul>
-        <button type="button">See project</button>
-      </article>
-    </section>
+      <section class="headline">
+        <article class="intro">
+          <div class="first_intro_column">
+            <h1 class="main_title">Hey There.<br />I'm Robert.</h1>
+            <h2 class="subtitle">I'm a Software Developer</h2>
+            <p class="desc">
+              I can help you build a product, feature or website. Look through
+              some of my work and experience! if you like what you see and have
+              a project you need coded, don't hesitate to contact me.
+            </p>
+          </div>
+          <div class="second_intro_column">
+            <ul>
+              <li>
+                <a href="https://github.com/Kanyorok">
+                  <span class="icon git"></span>
+                </a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/robert-mbugua-a8745093/">
+                  <span class="icon linked"></span>
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <span class="icon angel"></span>
+                </a>
+              </li>
+              <li>
+                <a href="https://twitter.com/CarlosMbugua">
+                  <span class="icon twitter"></span>
+                </a>
+              </li>
+              <li>
+                <a href="#">
+                  <span class="icon medium"></span>
+                </a>
+              </li>
+            </ul>
+          </div>
+        </article>
+        <div class="image_show">
+          <img src="icons/Illustration-header.svg" alt="back" />
+        </div>
+      </section>
 
-    <section class="center_content">
-      <article class="content_desc">
-        <h2>Profesional Art <br />Printing Data</h2>
-        <p>
-          A daily selection of privately personalized reads; no accounts or
-          sign-ups required. Has been the industry's standard.
-        </p>
-        <ul>
-          <li>
-            <a href="#">HTML</a>
-          </li>
-          <li>
-            <a href="#">BOOTSTRAP</a>
-          </li>
-          <li>
-            <a href="#">RUBY</a>
-          </li>
-        </ul>
-        <button type="button">See project</button>
-      </article>
-    </section>
+      <section class="works">
+        <header class="works_title">
+          <span> My Recent Works </span>
+          <hr />
+        </header>
```

Comment:
- [ ] I would suggest that you wrap this subheading in a `hx` tag and not a header tag because the `<header>` HTML element represents introductory content, typically a group of introductory or navigational aids. Fix this to improve the semantics and accessibility of your website.